pkg/gatherers/clusterconfig/gather_node_logs.go

```diff
@@ -119,7 +121,9 @@ func nodeLogString(ctx context.Context, req *rest.Request) (string, error) {
 	messagesToSearch := []string{
 		"E\\d{4} [0-9]{1,2}:[0-9]{1,2}:[0-9]{1,2}", //  Errors from log
 	}
-	return common.FilterLogFromScanner(scanner, messagesToSearch, true, func(lines []string) []string {
+	messagesRegex := regexp.MustCompile(strings.Join(messagesToSearch, "|"))
```

Comment:
It would be better to compile the regex once and not for every node.

Reply:
Oh yeah, for sure. I forgot to add the `TODO` here. I want to do something similar to what I did for `CollectLogsFromContainers`. Thanks for pointing it out.